packages/bulkapi/src/controllers/BulkApiWorker.ts

```diff
@@ -63,27 +82,51 @@ export class BulkApiWorker {
      * @param mode
      * @param depthLimit
      */
-    bulkRetrieve = async (nodeIdList: string[], mode: string, depthLimit: number): Promise<LionWebJsonChunk> => {
+    bulkRetrieve = async (nodeIdList: string[], mode: string, depthLimit: number): Promise<QueryReturnType<RetrieveResponse>> => {
         if (nodeIdList.length === 0) {
             return {
-                serializationFormatVersion: "2023.1",
-                languages: [],
-                nodes: [],
+                status: 200,
+                query: "",
+                queryResult: {
+                    success: true,
+                    messages: [{ kind: "EmptyIdList", message: "The list of ids is empty, empty chunk returned" }],
+                    chunk: {
```

Comment:
Could we define a constant called "EmptyChunk"?

Reply:
Done